assets/styles/modules/lists.scss

```diff
@@ -1,5 +1,6 @@
 // News overview
-.view-sharing-knowledge {
+.view-sharing-knowledge,
+.grid-container-view {
```

Comment:
why both classes? This implies a special treatment of `view-sharing-knowledge`. I think it is better to only have `grid-container-view` and then also assign this class to the sharing knowledge view via UI. So all views work the same way.